ui/components/app/fee-details-component/fee-details-component.js

```diff
@@ -107,7 +106,7 @@ export default function FeeDetailsComponent({
         justifyContent={JustifyContent.center}
         flexDirection={FlexDirection.Column}
       >
-        {supportsEIP1559 && shouldShow && (
+        {!hideGasDetails && (
```

Comment:
I think we can cleanup the preexisting code related to `hideGasDetails`. 

We can:
- rename `hideGasDetails` → `isHidden`
  this will help devs understand its true use, which is either hide or show everything
- remove checks in template and place this code above line 86 (current `return` line)
  ```
    if (hideGasDetails) {
      return null;
    }
  ```
  
  I don't believe there is a need to show the empty 
  ```
  <Box
        display={Display.Flex}
        alignItems={AlignItems.center}
        justifyContent={JustifyContent.center}
        flexDirection={FlexDirection.Column}
      >
  </>
  ```
  I checked with storybook + background color and there is no visible element in these cases

Reply:
@digiwand : I would avoid putting effort in changing current code executing. Change here is limited in scope to remove `supportsEIP1559` from `if` condition.

Reply:
ok this is fine. agreed we should limit scope with bug changes. I do think the readability of the code can be improved with these changes in a future PR